Content.Shared/SmartFridge/SharedSmartFridgeComponent.cs

```diff
@@ -0,0 +1,74 @@
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.SmartFridge
+{
+    public class SharedSmartFridgeComponent : Component
```

Comment:
Classes should never just be virtual now (this applies to all in the PR) and your IDE is probably giving you warnings as such.